### DISCUSSION TITLE
anyone could attend the Orientation Session which was scheduled for today 11.00 am

### DISCUSSION THREAD

**Discussion by Akhil-Tony** [github/india#244] [Students]
anyone ?

**Comment by Sumitkcr** [comment]
Not able to find the link... even site was not working

**Reply by TheSuranaverse** [reply]
**FUTURE READY SEPARATE CHANNEL**

For smooth communication & discussion!
Link: https://discord.gg/ceMXzhfaka

Thank you 🙏

**Comment by Akhil-Tony** [comment]
https://teams.microsoft.com/dl/launcher/launcher.html?url=%2F_%23%2Fl%2Fmeetup-join%2F19%3Ameeting_NzE0ZTUxZGItZWZjYy00MDQwLTk2Y2UtMDgwYjhjNGIxYmYz%40thread.v2%2F0%3Fcontext%3D%257b%2522Tid%2522%253a%25227870c433-4d93-4337-9aff-75a5d0074084%2522%252c%2522Oid%2522%253a%25227bea9672-c7bf-458d-8894-9f6a6fba38fb%2522%252c%2522IsBroadcastMeeting%2522%253atrue%257d%26btype%3Da%26role%3Da%26anon%3Dtrue&type=meetup-join&deeplinkId=99d98475-5789-4004-9757-51fea4c005be&directDl=true&msLaunch=true&enableMobilePage=true&suppressPrompt=true

**Comment by TheSuranaverse** [comment]
**FUTURE READY SEPARATE CHANNEL**

For smooth communication & discussion!
Link: https://discord.gg/ceMXzhfaka

Thank you 🙏
